README.md

```diff
@@ -209,3 +209,10 @@ These tools and add-ons make working with ZSS even easier.
 
 - [**color-name**](https://github.com/joppe/color-name)   
 CLI that parses the output of http://www.color-blindness.com/color-name-hue/ for easy converting of a hexadecimal color value to a name that can be used in ZSS to generate class names (e.g. `u-yellow--tangerine`).
+
+## Maintainer
+
+- Peter Benner ([@wpbenner](https://github.com/wpbenner/)) – [Zicht](https://github.com/zicht)
+- Annemieke van der Hoek ([@avanderhoek](https://github.com/avanderhoek/)) – [Zicht](https://github.com/zicht)
+- Jurg Roessen ([@Hangloozz](https://github.com/Hangloozz/)) – [Zicht](https://github.com/zicht)
```

Comment:
Ik zou de maintainers in alle gevallen in alfabetische volgorde zetten. En het is gebruikelijk om twee maintainers per bundle te hebben, kan dat ook hier? Of is er een goede reden om er drie te hebben?

Reply:
Ik begreep van @avanderhoek dat @Hangloozz heel veel in deze bundle werkt. Ik kan mezelf als maintainer eruit halen. Mocht dat nodig zijn.

Reply:
2 of 3 maintainers is mij om het even.  Gooi een muntje op zou ik zeggen :game_die:

Reply:
Ik heb mezelf als maintainer weg gehaald. Annemieke en Jurg werken meer aan deze bundle dus leek me dat handiger.